src/contracts/hollowdb.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  get,
+  put,
+  update,
+  remove,
+  evolve,
+  getKeys,
+  getKVMap,
+  updateOwner,
+  updateWhitelist,
+  updateVerificationKey,
+  updateRequirement,
+} from './functions';
+import {InvalidFunctionError} from './errors';
+import type {
+  ContractState,
+  ContractHandle,
+  EvolveInput,
+  GetInput,
+  GetKVMapInput,
+  GetKeysInput,
+  PutInput,
+  RemoveInput,
+  UpdateInput,
+  UpdateOwnerInput,
+  UpdateRequirementInput,
+  UpdateVerificationKeyInput,
+  UpdateWhitelistInput,
+} from './types';
+
+export type HollowState = ContractState<{circuits: ['auth']; whitelists: ['put', 'update']}>;
```

Comment:
Each contract specifies the circuits and whitelists that they would like to use. In this case, `auth` circuit is used with `put` and `update` whitelists.